src/main/java/app/dao/impl/LogsDaoImpl.java

```diff
@@ -1,55 +1,70 @@
-import app.entities.namespace.LogsNamespace;
-import app.exceptions.EntityNotFoundException;
-import org.springframework.stereotype.Repository;
+import java.sql.*;
+import java.util.Calendar;
 
-import java.util.Date;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.ListIterator;
+public class LogsDaoImpl {
+    private final String HOST = "jdbc:mysql://localhost:3306/logs";
+    private final String USERNAME = "root";
+    private final String PASSWORD = "123";
+    private Connection connection;
+    PreparedStatement preparedStatement = null;
 
-@Repository
-public class LogsDaoImpl implements LogsDao {
-
-    private static List<Logs> logsList = new LinkedList<>();
-
-    @Override
-    public synchronized List<Logs> getLogFor(LogsNamespace logsNamespace) {
-        Date boundaryDate = logsNamespace.getBoundaryDate();
-        LinkedList<Logs> resultLogsList = new LinkedList<>();
-
-        ListIterator<Logs> logsIterator
-                = logsList.listIterator(logsList.size());
-        Logs previous;
-        while (logsIterator.hasPrevious()) {
-            previous = logsIterator.previous();
-            if (previous.getDate().before(boundaryDate)) {
-                break;
-            }
-            resultLogsList.add(previous);
+    public LogsDaoImpl() {
+        try {
+            connection = DriverManager.getConnection(HOST, USERNAME, PASSWORD);
+            preparedStatement = connection.prepareStatement("INSERT INTO logs (projectId,employeeId,time,comment,date) " +
+                    "values (?,?,?,?,?)");
+            preparedStatement = connection.prepareStatement("delete from logs where id=?");
+        } catch (SQLException e) {
+            e.printStackTrace();
         }
-        return resultLogsList;
     }
 
-    @Override
-    public synchronized List<Logs> getAll() {
-        return logsList;
+    public String getAll() {
+        Logs logs = new Logs();
+        String query = "select*from logs";
+        try {
+            Statement st = connection.createStatement();
+            ResultSet resultSet = st.executeQuery(query);
+            while (resultSet.next()) {
+                logs.setId(resultSet.getInt(1));
+                logs.setProject(resultSet.getInt(2));
+                logs.setEmployee(resultSet.getInt(3));
+                logs.setTime(resultSet.getInt(4));
+                logs.setComment(resultSet.getString(5));
+                logs.setDate(resultSet.getDate(6));
+                System.out.println(logs.toString());
+            }
+            st.close();
+        } catch (SQLException e) {
+        }
+        return logs.toString();
     }
 
-    @Override
-    public synchronized void save(Logs logs) {
-        logsList.add(logs);
+    public void save() {
+        try {
+            preparedStatement.setInt(1, 2);
+            preparedStatement.setInt(2, 3);
+            preparedStatement.setInt(3, 8);
+            preparedStatement.setString(4, "Добавить рисунок");
```

Comment:
hardcoded values? you should take them from your rest serves

Reply:
corrected